conf-files/UPHL_viralrecon_params.yml

```diff
@@ -0,0 +1,16 @@
+platform: 'illumina'
+protocol: 'amplicon'
+primer_set: 'artic'
+kraken2_db: '/Volumes/NGS/Bioinformatics/pooja/kraken2_human.tar.gz'
+kraken2_db_name: 'human'
```

Comment:
I didn't realize that viralrecon used the compressed kraken2 file.

Reply:
Cecret and Grandeur also use Kraken2 databases. Does this one use a specific one? Do you know how often it needs to be updated? (Cecret never needs to be updated, Grandeur is updated with every refseq release)

Reply:
It doesn't need to be updated as such. I just have been using this since I installed the pipeline. Viralrecon does come with a default kraken database but it was giving me errors at some point so I just started using my own downloaded version. I can surely change it to something that already gets used by Cecret or Grandeur.

Reply:
I think we've got ~15 kraken2 databases on the NAS at the moment. I don't know when we need to start consolidating.

Reply:
Yep! I searched on the NAS and wasn't sure which databases were complete and recent.